servicenow/6.7/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,85 @@
+= ServiceNow Connector 6.7 - Mule 4
+:page-aliases: connectors::servicenow/servicenow-about.adoc, connectors::servicenow/servicenow-to-use.adoc, connectors::servicenow/servicenow-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+
+Anypoint Connector for ServiceNow (ServiceNow Connector) provides connections between Mule runtime engine (Mule) and ServiceNow apps.
+Use the ServiceNow operations with the custom ServiceNow tables, along with any operations available through the installed plugins.
+
+For compatibility information and fixed issues, see the ServiceNow Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements in a Mule flow and global elements
+* How to create a Mule app using Design Center or Anypoint Studio
+
+Before creating an app, you must have access to the ServiceNow target resource and Anypoint Platform.
+
+
+== Audience
+
+* New user
++
+To create your Mule app,
+see xref:servicenow-connector-studio.adoc[Studio Configuration].
+* Advanced user
++
+Read the xref:servicenow-connector-xml-maven.adoc[XML and Maven Support]
+and xref:servicenow-connector-examples.adoc[Examples] topics.
+The Examples topic provides one or more use cases for using the connector.
```

Comment:
```suggestion
* New users, read
** xref:servicenow-connector-studio.adoc[Use Studio to Configure the Connector] to create your Mule app
* Advanced users, read:
** xref:servicenow-connector-xml-maven.adoc[XML and Maven Support]
** xref:servicenow-connector-examples.adoc[Examples], which provides one or more use cases for the connector
```

Reply:
Also, I would move the Audiences section afterwards the Common Use Cases for the Connector